packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -1,69 +1,21 @@
-/***********************************************************
- * This file holds the class definitions for the different *
- * components (question types) including base types        *
- ***********************************************************/
-
-const nunjucks = require('nunjucks');
-
-// todo:
-// test
-// refactor
-// jsdoc
-
 class Question {
 	title;
 	question;
 	description;
 	type;
 	fieldName;
+	validators = [];
```

Comment:
These fields cover what we need for question 1 but longer term we'll also need the following:
`hint` (string)
`bulletPoints` (array of strings)
`details` (object with a `title` and `text` field)

Reply:
Can you update the ticket?